pages/code-review.md

```diff
@@ -90,22 +124,23 @@ Before seeking a review, you should be able to check off each of the following:
     - [ ] Code follows the standards laid out in this playbook
     - [ ] Passes all existing automated tests
     - [ ] New functions include new tests
-    - [ ] New functions are documented (with a description, list of inputs, and
-         expected output)
+    - [ ] New functions are documented (with a description, list of inputs,
+          and expected output)
     - [ ] Placeholder code is flagged
     - [ ] Visually tested in supported browsers and devices
-    - [ ] Project documentation has been updated (including the "Unreleased" section of
-          the CHANGELOG)
+    - [ ] Project documentation has been updated (including the "Unreleased"
+          section of the CHANGELOG)
 
 ### Opening a PR
 
 Once your code meets the checks above, open a Pull Request for your new
-feature or bugfix.Remember that the more information you provide to reviews,
+feature or bugfix. Remember that the more information you provide to reviews,
```

Comment:
In looking at that sentence more closely (I just added the space after the period), yes, looks like it was incorrect.